tests/instances.tcl

```diff
@@ -391,6 +391,11 @@ proc check_leaks instance_types {
 
 # Execute all the units inside the 'tests' directory.
 proc run_tests {} {
+    set sentinel_fd_leaks_file "sentinel_fd_leaks"
```

Comment:
i would like to move that file into the `tests/sentinel/tmp/` folder.

Reply:
Actually, this file would be created in that path by the current code:
WARNING: sentinel test(s) failed, there are leaked fds in sentinel,
        see output file: /data/c/redis/tests/sentinel/tmp/sentinel_fd_leaks for more details.